app/interfaces/account.interface.ts

```diff
@@ -0,0 +1,11 @@
+export interface Account {
+    account_id?: string;
+    accountId?: string;
```

Comment:
why account_id and accountId

Reply:
Because I might have used accountId somewhere rather than account_id, and account_id somewhere else, I want to use one interface for all of them